src/developTest/java/mServer/crawler/sender/newsearch/ZdfDownloadDTODeserializerTest.java

```diff
@@ -0,0 +1,52 @@
+package mServer.crawler.sender.newsearch;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00120)